internal/ansible/flags/flag.go

```diff
@@ -82,6 +83,11 @@ func (f *Flags) AddTo(flagSet *pflag.FlagSet) {
 		":8080",
 		"The address the metric endpoint binds to",
 	)
+	flagSet.StringVar(&f.ProbeAddr,
+		"health-probe-bind-address",
+		":8081",
```

Comment:
Port number needs to stay the same too
```suggestion
		":6789",
```

Reply:
Since users can customize it as they wish then, in POV it is a change no breaking change. I'd prefer we keep all projects aligned. I tried to add it in the fragment. Let me know wdyt.

Reply:
In order to keep the same behavior as before this PR, a user would have to modify any operator code that depends on the current health probe being at [the default address `0.0.0.0:6789`](https://github.com/operator-framework/operator-sdk/pull/4326/files#diff-8f8a3e7e238207f04c7258c16670966e1e72ca063527fcd0ec6429c475d35cd0L108), which means changing the port is a breaking change.